packages/cozy-app-publish/README.md

```diff
@@ -208,3 +208,21 @@ We apply the semver convention:
 - If the release brings only bug fixes, then we should create a patch version.
 - If the release brings at least one feature, then we should create a minor version.
 - If the release changes its application's permissions, we should create a minor version even if this release is for bug fixes only.
+
+### Hot Fix
+
+If you need to qucikly fix a bug in production, then you have to:
+- fetch the latest branch release 
+- create a new release branch from there (since this is a bug fix, this is a patch version)
+- fix the bug 
+- Make a beta
+- Publish
+- Merge it on master
+
+Exemple: You have to hot fix Drive. The currection `stable` version of Drive is : 1.45.0. Master is on 1.47.0. 
+Then you need to fetch release/1.45.0 
```

Comment:
⚠️ since there is no blank line between those lines, everything will be collapsed into one single line.